modin/pandas/series.py

```diff
@@ -803,12 +803,6 @@ def corr(self, other, method="pearson", min_periods=None):  # noqa: PR01, RT01,
             )
         )
 
-    def count(self, level=None):  # noqa: PR01, RT01, D200
```

Comment:
this has extra params `numeric_only` and `axis`: https://github.com/modin-project/modin/runs/7982177596?check_suite_focus=true